components/Cards/src/MDCCardCollectionCell.m

```diff
@@ -477,6 +500,53 @@ - (void)tintColorDidChange {
   [self setImageTintColor:self.tintColor forState:MDCCardCellStateNormal];
 }
 
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
+  if (self.layer.shapeGenerator) {
+    if (CGPathContainsPoint(self.layer.shapeLayer.path, nil, point, true)) {
+      return self;
+    } else {
+      return nil;
+    }
+  }
+  return [super hitTest:point withEvent:event];
```

Comment:
This is a UX question in my mind, which is actually interesting because both perspectives could seem valid. My initial thought was that as a user who doesn't understand the invisible rectangular bounds that each component/widget consists of, I would expect that if I tap something I can see, then it would invoke a tap, whereas if I tap outside of it, it wont tap.
However, I am aware that when setting the `cornerRadius` of a cell to create a circular cell, then `UIKit` doesn't change the `hitTest` and therefore any tap in the cell (even outside the circle) is seen as a tap. Therefore I might lean towards what Apple does as a default, because they probably have thought this through more than I. I will remove this for now, but open to bring it back if others think strongly the other way.